SMLHelper/BiomeThings/FloatingOrigin.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Based on the Unity Wiki FloatingOrigin script by Peter Stirling
+// URL: http://wiki.unity3d.com/index.php/Floating_Origin
+
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+internal class FloatingOrigin : MonoBehaviour
```

Comment:
No namespace?

Reply:
I'm going to remove this system, as it's very buggy.